docs/install_guides/configurations-for-ncs2.md

```diff
@@ -11,68 +11,29 @@
 
 Once you have OpenVINO™ Runtime installed, follow these steps to be able to work on NCS2:
 
-1. Go to the install_dependencies directory:
-   ```sh
-   cd <INSTALL_DIR>/install_dependencies/
-   ```
-2. Run the `install_NCS_udev_rules.sh` script:
-   ```
-   ./install_NCS_udev_rules.sh
-   ```
-3. You may need to reboot your machine for this to take effect.
-
-You've completed all required configuration steps to perform inference on Intel® Neural Compute Stick 2. 
-Proceed to the [Get Started Guide](@ref get_started) section to learn the basic OpenVINO™ workflow and run code samples and demo applications.
-
-@sphinxdirective
-
-.. _ncs guide raspbianos:
-
-@endsphinxdirective
-
-## Raspbian OS
-
 1. Add the current Linux user to the `users` group:
    ```sh
    sudo usermod -a -G users "$(whoami)"
    ```
-   Log out and log in for it to take effect.
-2. If you didn't modify `.bashrc` to permanently set the environment variables, run `setupvars.sh` again after logging in:
+2. Go to the install_dependencies directory:
    ```sh
-   source /opt/intel/openvino_2022/setupvars.sh
+   cd <INSTALL_DIR>/install_dependencies/
    ```
-3. To perform inference on the Intel® Neural Compute Stick 2, install the USB rules running the `install_NCS_udev_rules.sh` script:
-   ```sh
-   sh /opt/intel/openvino_2022/install_dependencies/install_NCS_udev_rules.sh
+3. Copy the `97-myriad-usbboot.rules` file to the udev rules directory:
    ```
-4. Plug in your Intel® Neural Compute Stick 2.
-
-5. (Optional) If you want to compile and run the Image Classification sample to verify the installation of OpenVINO, follow the steps below.
+   sudo cp 97-myriad-usbboot.rules /etc/udev/rules.d/
```

Comment:
do you think we can inline the content of usbboot rules like here https://github.com/openvinotoolkit/openvino/wiki/BuildingForRaspbianStretchOS#for-linux-raspbian-stretch-os and drop this file as well?

Reply:
Decided to keep this file for 2022.3.* LTS releases